doit/doit_cmd.py

```diff
@@ -52,18 +52,28 @@ class DoitMain(object):
                  Strace, TabCompletion, ResetDep)
 
     def __init__(self, task_loader=None,
-                 config_filenames='doit.cfg',
+                 config_filenames=['pyproject.toml', 'doit.cfg'],
```

Comment:
This is to align with the existing behavior of `ConfigParser.read`, which appears to be FIFO... I verified it, as i didn't see it clearly [documented](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.read). but makes sense.

Reply:
Not sure what you mean by FIFO. which file has priority, first or last?

Reply:
Last-read wins. It seems to be the convention used by [isort](https://pycqa.github.io/isort/docs/configuration/config_files/), [pytest](https://docs.pytest.org/en/6.2.x/customize.html#configuration-file-formats), while `black` _only_ supports `pyproject.toml`.